lts/snap/snapcraft.yaml

```diff
@@ -5,7 +5,7 @@ icon: assets/icon.png
 grade: stable
 
 adopt-info: powershell
-base: core18
+base: core20
```

Comment:
As mentioned in https://github.com/PowerShell/PowerShell-Snap/issues/87#issuecomment-1216799996 it might be advisable to skip `core20` and jump directly to `core22`.

Reply:
Thanks for the thorough explanation of the benefits of upgrading to core22. I'll discuss this with the team.

Reply:
The guidance we've received is that core20 is the recommended version right now as it the most stable release. We'll update to core20 first and then can evaluate core22 as needed.